README.md

```diff
@@ -6,106 +8,64 @@
 
 1. What is the first step for installing Dart on a Windows machine?
 
-A) Install Homebrew
-B) Download the Dart SDK
-C) Update your PATH
-D) Run Dart Doctor
-
+'C) Update your PATH '
```

Comment:
DOWNLOADING DART SDK IS ALWAYS THE FIRST STEP

Reply:
B downloading dart sdk

Reply:
B downloading the SDK

Reply:
B Download the Dart SDK

Reply:
B) Download the Dart SDK

Reply:
Download the Dart SDK